### PR TITLE
Update platforms.md

### PR DESCRIPTION
**Pull request by gveres**

### CODE REVIEW

File: reference/platforms.md [hunk 1]
```diff
@@ -10,9 +10,9 @@
 | DAM        | Digital Asset Management              |
 | MA         | Marketing Automation                  |
 | POS        | Point of Sale                         |
-| Search     | and Findability                       |
+| Search     | Search and Findability                |
 | Recommendation     | Content or product recommendation                       |
-| Customer   | Service                               |
+| Enterprise Help Desk   | Customer Service Platform |
```
Comment (ramonsnir) [inline reference/platforms.md:15]: I think this table is meant to explain, among other things, this diagram's terminology: https://netlify.slack.com/archives/D06H8J2NBNF/p1717086989018409

I don't see "Customer" or "Enterprise Help Desk" but I do see "Customer Service". Perhaps this row should be "Customer Service" on the left side and an explanation on the right side?

File: reference/platforms.md [hunk 2]
```diff
@@ -10,9 +10,9 @@
 | DAM        | Digital Asset Management              |
 | MA         | Marketing Automation                  |
 | POS        | Point of Sale                         |
-| Search     | and Findability                       |
```
Comment (smaida) [inline reference/platforms.md:13]: Does it make sense for this to refer to Discoverability in addition to or in place of Findability? We've been an Algolia partner for years, and I've generally found Discoverability to be a common term in search and discovery conversations, with search and Findability being more synonymous. Discovery is also used by Gartner when discussing the category.

I always refer back to this old post from Seth Godin. 
https://seths.blog/2014/04/search-vs-discovery/

"Discovery, on the other hand, is what happens when the universe (or an organization, or a friend) helps you encounter something you didn't even know you were looking for."

Modern search platforms aid not only in Search (Findability) but also in Discoverability.

Comment (gveres) [reply reference/platforms.md:13]: Good point - updated, thanks!

File: reference/platforms.md (hunk 1 above)
Comment (gveres) [reply reference/platforms.md]: How about the suggested change? I was struggling with this as it does not seem to have a widely accepted acronym, so a bit weird to "resolve" this...

**Review by adampnielsen — APPROVED** [review]

**Pull request merged**